examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
 ## Templates Guidline
 
-These templates are a base for the new code examples.
+This folder contains templates for new code examples. Each new code example should be created using the template from this folder. We have templates for examples made in React, Angular, Vue, and of course Vanilla JavaScript.
+
+#### Prerequisite
+To work with code examples (and Handsontable development in general) use at least Node version `15.9.0`. If you have Node Version Manager installed, you can just type in the terminal `nvm use`.
 
 ### Creating a new example
 
-All new code examples should be added to the `/examples/next` directory. Based on them you can create versioned code examples.
+In this guideline, we will walk you through the process of creating new code examples. 
+
+> If you need some inspiration, what can be done with code examples, go ahead see existing examples in `examples/next/docs/` to get inspired!
```

Comment:
WDYT
If you want to get inspired before moving forward, see the already existing code examples in the folder `examples/next/docs/`.